apps/src/StudioApp.js

```diff
@@ -3,6 +3,7 @@
 
 var parseXmlElement = require('./xml').parseElement;
 var utils = require('./utils');
+var dropletUtils = require('./dropletUtils');
```

Comment:
It is neutral. The functions were all previously in utils (which was included). In a sense, I decreased code size for the apps that don't yet support droplet (calc, eval, etc.) since they were all including utils. But for those that do, they will include these utility functions (which are very small compared to the whole of dropet/ace/interpreter)